### ISSUE TITLE
Cyclic learning rate added to Pytorch!

### ISSUE THREAD

**Issue by thomasjpfan** [skorch-dev/skorch#448]
In the future, we can remove our version and adapt the official version: https://github.com/pytorch/pytorch/commit/8635078d9ed47266e6df89ad5ef887b598a76f25

**Comment by ottonemo** [comment]
This sounds like good news :) 
As far as I can see there is no 'batch step' and it uses the epoch instead. This differs from other implementation and changes behaviour, right?

**Comment by thomasjpfan** [comment]
Their step means batch_step, which needs to be called once per batch. We would need to treat cyclicLR as a special case in our callback.

**Comment by thomasjpfan** [comment]
https://github.com/pytorch/pytorch/releases/tag/v1.1.0

Now that CyclicLR is merged, how should we go about integrating it? We would need to deprecate our version and start to use the PyTorch version. The PyTorch version is very similar to ours (because it is based on a PR I made for it).

**Comment by BenjaminBossan** [comment]
I personally haven't worked much with the learning rate schedulers, you know much better how to best to integrate them. My proposal would be to deprecate the existing one, adding a detailed message with an example of how to use the new CyclicLR.

Furthermore, we should add the requirement for torch 1.1 and probably make a new release once the open PRs are finished.

**Comment by BenjaminBossan** [comment]
@thomasjpfan This was fixed with #473 or is there anything left?

**Comment by thomasjpfan** [comment]
Nope this issue is fixed.
